src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx

```diff
@@ -30,21 +33,49 @@ class ConfirmationPage extends React.Component {
     this.setState({ isExpanded: !this.state.isExpanded });
   }
 
+  makeList(arr) {
+    if (arr && arr.length) {
+      return (
+        <ul className="claim-list">
+          {
+            arr.map((d, i) => {
+              return (<li key={i}>{d}</li>);
+            })
+          }
+        </ul>
+      );
+    }
+    return null;
+  }
 
   render() {
     const form = this.props.form;
     const response = this.props.form.submission.response
       ? this.props.form.submission.response.attributes
       : {};
     const name = form.data.veteranFullName;
+    const benefits = form.data['view:selectedBenefits'];
+    const benefitsRelinquished = _.get('data.view:benefitsRelinquishedContainer.benefitsRelinquished', form);
 
     const docExplanation = this.state.isExpanded
       ? (<div className="usa-accordion-content">
-        <p>In the future, you might need a copy of your DD 2863 (National Call to Service (NCS) Election of Options).</p>
+        <p>In the future, you might need:</p>
+        <ul>
+          <li>Your reserve kicker</li>
+          <li>Documentation of additional contributions that would increase your monthly benefits.</li>
+        </ul>
         <p>Documents can be uploaded using the <a href="https://gibill.custhelp.com/app/utils/login_form/redirect/account%252">GI Bill site</a>.</p>
```

Comment:
Is the `%252` necessary for some reason? I know it's not a new change, but...it's odd.